plugins/CoreHome/vue/src/ReportingMenu/ReportingMenu.vue

```diff
@@ -12,13 +12,18 @@
       :aria-label="translate('CoreHome_MainNavigation')"
     >
       <li
-        class="menuTab"
         role="menuitem"
         v-for="category in menu"
-        :class="{ 'active': category.id === activeCategory }"
+        :class="`menuTab ${category.id === activeCategory ? 'active' : null} ${category.id}`"
```

Comment:
Outputting the category ID may need some sanitisation in case it's not CSS class name compatible.

Reply:
What is the category id as class name actually good for? You could also use a custom html attribute for that. That would allow any value and could be used in CSS as well.

Reply:
Yeah, it's to apply a specific styling. We can use a custom data attribute, or convert the ID to a CSS-able string. The result and use is the same in the end. I guess a data attribute can stomach more content than a CSS class name.

Reply:
replaced with a data attribute